sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/NotSupportedAssetFileTypeException.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion
+{
+    /// <summary>
+    /// An Exception thrown during an attempt to provide an unsupported asset file type in an asset conversion operation
+    /// </summary>
+    public class NotSupportedAssetFileTypeException : Exception, ISerializable
```

Comment:
Do we need the Serializable attribute?

Reply:
Not for this use case to my knowledge

Reply:
@tg-msft thoughts here? this was based on your feedback

Reply:
I don't think it's required - you can leave what you have.

Reply:
This is required according to CA2237 - trying to figure out why it is not firing. /cc @pakrym 

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2237

Reply:
It seems like the warning is only triggered if net461 is include in the list of target frameworks. I think we can follow up separately to determine whether we need to add this attribute for all of the T2 exception types.